src/App.js

```diff
@@ -9,6 +9,7 @@ import SingleArticle from "./components/SingleArticle";
 function App() {
   const [topicList, setTopicList] = useState([]);
   const [searchParams, setSearchParams] = useSearchParams({});
+  const [user, setUser] = useState("grumpy19");
```

Comment:
How about you put your user in state? Then you won't need to be passing it down to multiple components through props.